src/js/components/inspectors/MultiValueProperty.test.jsx

```diff
@@ -0,0 +1,239 @@
+/* eslint no-unused-expressions:0,new-cap:0 */
```

Comment:
not sure if I should get these tests to work or how do I run them?

Reply:
the tests are stale so ignore them!